.babelrc

```diff
@@ -1,12 +1,3 @@
 {
-  "presets": [
-    [
-      "@babel/preset-react",
-      {
-        "runtime": "automatic"
-      }
-    ],
-    "@babel/preset-env"
-  ],
```

Comment:
Removed these lines since Parcel already loads these presets so was complaining for them to be dupe.